cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+
+Strong emphasis, aka bold, with double **asterisks**.
+```
+
+The text contained within headings can be emphasized.
+
+Both italicized and bold cannot be applied to the same text.
+
+#### Code and Syntax Highlighting
+
+Texted can be highlighted as code, when encased without spaces by backticks.
+This MUST not contain line breaks.
+
+Example:
+```md
+Inline `code` has `back-ticks around` it.
+```
+
+The text contained within headings or emphasis cannot be highlighted as code.
+
+
+#### Unordered Lists
+
+// todo
+
+#### Line Breaks / Paragraphs
+
+To create paragraphs, use a blank line to separate one or more lines of text.
+
+Examples:
+```md
+Here's a line for us to start with.
+
+This line is separated from the one above by two newlines, so it will be a *separate paragraph*.
+
+This line is also a separate paragraph, but...
+This line is only separated by a single newline, so it's a separate line in the *same paragraph*.
+```
+
+### Best Practices
+- when hash doesnt match
+- when unrecognised formatting/style
+- Keep lines left-aligned without unneeded whitespace or tabs
+- tabs vs spaces
+
+## Rationale: how does this CIP achieve its goals?
+
+Just like CIP-1694, we aim to define a minimal viable set of standards to ensure governance can be achieved.
+
+### Sentences
+- improve the readability of diff views
+
+### Versioning
+- why not build in options in hashalgo (like CIP100)
+- why a whole replacement?
+
+### File Type
+- just simple and easy
+
+### Hashing
+- we choose blake2b-256 as it is a standard for hashing in Cardano
+- standard, lots of tools already use it 
+
+### Storage
+- The constitution is the most important document for Cardano (?), ensuring its permissionless accessibly is paramount
+- Storing each iteration immutably is just as important, to allow access of past constitutions
+- IPFS is well known already with good tooling support -> improve accessibility
+
+### Rich Text Formatting
+- why not all markdown?
+- wanted to keep a bare minimum spec, no images or videos
+- also dont want people to be able to attack consumers by attacks utilizing the HTML
+- wanted to allow standard markdown libs to work
+
+## Open Questions
+- [x] How can we support multi-languages?
+  - The Cardano constitution will be in English, but we will add best practice guidelines via [Best Practices](#best-practices).
+- [ ] Should we specify any standardization for the proposal policy?
+- [x] How can we add page breaks?
+  - We wont, instead we will prioritize a minimum set of rich text formatting. We can provide some guidance via [Best Practices](#best-practices).
+- [x] Do we want a mechanism for specifying authors? (similar to CIP-100)
+  - No, as CIP-100 compliant metadata can be supplied at time of constitution update.
+- [ ] What should we name the constitution file? we could embed some nice naming or metadata.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] This standard is followed for the interim Cardano constitution
```

Comment:
```suggestion
- [ ] This standard is followed for the interim Cardano Constitution
```
Last nitpick this time around I promise: there will be only one interim constitution so it should be capitalised as proper noun.  It is ambiguous (in my understanding) whether Interim should also be capitalised... perhaps having political implications if it were so & therefore better to leave in lowercase. 😜

Reply:
ALL NITPICKS APPRECIATED 😎

Reply:
thanks... @Ryun1 do you know if the working groups would prefer "interim" by itself as a relatively insignificant modifier, or including it capitalised in the title as "Interim Cardano Constitution"?

Reply:
I dont think there is official language defined, happy to use what you suggest

Reply:
There will be only one Interim Constitution, so it would be grammatically correct to capitalise it. 🤓